src/test/java/uk/gov/ons/ctp/integration/rhsvc/message/impl/UacEventReceiverImplIT_Test.java

```diff
@@ -136,6 +147,9 @@ private UacEvent createUAC(String qid, EventTopic topic) {
     uac.setActive(true);
     uac.setQid(qid);
     uac.setCaseId("c45de4dc-3c3b-11e9-b210-d663bd873d93");
+    uac.setSurveyId("d45de4dc-3c3b-11e9-b210-d663bd873d93");
```

Comment:
there is a ready made JSON file that you could load with `FixtureHelper` for this

Reply:
Done